2022-01-04 02:23:30,326 INFO    MainThread:92143 [wandb_setup.py:_flush():71] setting env: {}
2022-01-04 02:23:30,327 INFO    MainThread:92143 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-04 02:23:30,327 INFO    MainThread:92143 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-04 02:23:30,328 INFO    MainThread:92143 [wandb_init.py:_log_setup():371] Logging user logs to /home/puppy/projects/BCI/wandb/run-20220104_022330-1qq7ptla/logs/debug.log
2022-01-04 02:23:30,328 INFO    MainThread:92143 [wandb_init.py:_log_setup():372] Logging internal logs to /home/puppy/projects/BCI/wandb/run-20220104_022330-1qq7ptla/logs/debug-internal.log
2022-01-04 02:23:30,329 INFO    MainThread:92143 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7efebca96190>
2022-01-04 02:23:30,330 INFO    MainThread:92143 [wandb_init.py:init():404] calling init triggers
2022-01-04 02:23:30,330 INFO    MainThread:92143 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-04 02:23:30,331 INFO    MainThread:92143 [wandb_init.py:init():460] starting backend
2022-01-04 02:23:30,331 INFO    MainThread:92143 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-04 02:23:30,368 INFO    MainThread:92143 [backend.py:ensure_launched():216] starting backend process...
2022-01-04 02:23:30,404 INFO    MainThread:92143 [backend.py:ensure_launched():221] started backend process with pid: 92236
2022-01-04 02:23:30,409 INFO    MainThread:92143 [wandb_init.py:init():469] backend started and connected
2022-01-04 02:23:30,620 INFO    MainThread:92143 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-01-04 02:23:30,621 INFO    MainThread:92143 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-04 02:23:30,622 INFO    MainThread:92143 [wandb_init.py:init():533] updated telemetry
2022-01-04 02:23:30,632 INFO    MainThread:92143 [wandb_init.py:init():563] communicating current version
2022-01-04 02:23:35,633 INFO    MainThread:92143 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-04 02:23:38,912 INFO    MainThread:92143 [wandb_init.py:init():606] starting run threads in backend
2022-01-04 02:23:43,928 INFO    MainThread:92143 [wandb_run.py:_console_start():1810] atexit reg
2022-01-04 02:23:43,929 INFO    MainThread:92143 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-04 02:23:43,931 INFO    MainThread:92143 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-04 02:23:43,933 INFO    MainThread:92143 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-04 02:23:43,934 INFO    MainThread:92143 [wandb_init.py:init():633] run started, returning control to user process
2022-01-04 02:31:45,453 INFO    MainThread:92143 [wandb_init.py:_pause_backend():284] pausing backend
2022-01-04 02:31:45,454 INFO    MainThread:92143 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-04 02:32:47,642 INFO    MainThread:92143 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-01-04 02:32:47,646 INFO    MainThread:92143 [wandb_run.py:_restore():1752] restore
